### ISSUE TITLE
Package or namespace load failed for ‘argoFloats’ in loadNamespace(package, lib.loc):

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#81]
Hi Team,

When I try to clean and rebuild the package, I receive the following error:

```R
Error : package or namespace load failed for ‘argoFloats’ in loadNamespace(package, lib.loc):
 cyclic namespace dependency detected when loading ‘argoFloats’, already loading ‘argoFloats’
Error: unable to load R code in package ‘argoFloats’
```

**Comment by dankelley** [comment]
I cannot reproduce this, with the "develop" branch commit 14388ab11026bb48743e5d3f75b14098f256543a , on my mac (catalina, beta), R 4.0. 

I wonder if @richardsc can reproduce it?

**Comment by dankelley** [comment]
@j-harbin -- maybe you can state the commit code for the version you're testing.

**Comment by dankelley** [comment]
I did a 'git pull' and got @j-harbin's new updates, and I get error as follows. I suggest removing the file and retesting.
```R
==> devtools::document(roclets = c('rd', 'collate', 'namespace', 'vignette'))

Updating argoFloats documentation
Loading argoFloats
Error in getIndex(file = "merged") (from duplicateProfiles.R#2) : could not find function "getIndex"
Calls: suppressPackageStartupMessages ... withr_with_dir -> force -> source_many -> source_one -> eval -> eval
Execution halted

Exited with status 1.
```

**Comment by j-harbin** [comment]
Whoops! I accidentally saved my profileDuplicate.R file into the R folder instead of my sandbox. All is good now!

Thanks @dankelley :)

**Comment by dankelley** [comment]
@j-harbin can you remove the file and commit to "develop", adding a note to that effect (with the hash code) here?  Thanks!

**Comment by j-harbin** [comment]
The commit to fix this issue can be found at commit #e026c20 in the develop branch.

**Comment by dankelley** [comment]
Thanks!
